src/platform_impl/macos/mod.rs

```diff
@@ -69,6 +70,7 @@ impl Window {
         pl_attribs: PlatformSpecificWindowBuilderAttributes,
     ) -> Result<Self, RootOsError> {
         let (window, _delegate) = UnownedWindow::new(attributes, pl_attribs)?;
+        menu::initialize();
```

Comment:
This doesn't seem to need to interact with winit internals at all. Is there some reason this can't be released as an separate crate?

Reply:
I guess the primary reason for that is that it's something essential for macOS and other windowing toolkits are having such things in them, like glfw. I'm not sure that extracting to other crate will make sense, given that literally every macOS application you'll be creating will require that crate, thus moving that into winit kind of makes sense, since it's less code to maintain and worry about.

Having to always add dependency to each windowing app in cross platform toolkit like winit for just macOS seems like a bad design to me.